package.json

```diff
@@ -30,12 +30,12 @@
     "all": true
   },
   "dependencies": {
-    "@dashevo/dapi-client": "0.8.0-dev.9",
-    "@dashevo/dapi-grpc": "0.12.0-dev.9",
+    "@dashevo/dapi-client": "dashevo/dapi-client#platform-services",
```

Comment:
Don't we need to change this one back after we merge https://github.com/dashevo/dapi-client/pull/98/ ?